playbooks/roles/wireguard/templates/instructions.md.j2

```diff
@@ -35,3 +41,34 @@ WireGuard is currently only available for Linux, but [cross-platform](https://ww
 
 #### A note on DNS configuration ####
 Each client configuration profile includes a `DNS` command that uses resolvconf to direct DNS traffic to the dnsmasq server that is available via the WireGuard encrypted interface at `{{ dnsmasq_wireguard_ip }}`. Although resolvconf is a common utility, you may need to use `PostUp`/`PostDown` with a different command for your distribution or particular network configuration.
+
+
+<a name="macos"></a>
+### macOS ###
+**WARNING**: Wireguard macOS client is in early stages of development and still considered experimental. It may be unstable or buggy.
+
+
+1. Install [Homebrew](https://brew.sh/), if you haven't already.
+1. Install Wireguard Tools using Homebrew:
```

Comment:
One of
1. the WireGuard tools
1. WireGuard tools
1. `wireguard-tools`

I'd prefer the first one, but your choice.